Source/PLCrashLogWriter.m

```diff
@@ -1358,6 +1384,18 @@ plcrash_error_t plcrash_log_writer_write (plcrash_log_writer_t *writer,
         plcrash_writer_write_binary_image(file, &image->macho_image);
     }
 
+    /* Try to read crash reporter annotations */
+    struct crashreporter_annotations_t *annotations = NULL;
+    image = plcrash_async_image_containing_address(image_list, siginfo->bsd_info->address);
```

Comment:
Is it redefined? Does it hide some other variable?

Reply:
This var is used inside the loop above. It's safe to just reuse it here